dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,258 @@
+# Setting up a Mac-based development environment #
+
+## Initial Account Setup (as Administrator) ##
+
+For this section, log in as your Administrator account.
+
+### Account Permissions ###
+
+If you have a Standard user account, you’ll need to allow it access to run
```

Comment:
I don't think "Standard" needs to be capitalized here.

Reply:
Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4